module_utils/ansible_nailgun_cement.py

```diff
@@ -6,7 +6,7 @@
 import yaml
 
 from nailgun.config import ServerConfig
-import nailgun.entities
+from nailgun import entities
 
```

Comment:
I would actually prefer it be `from nailgun.entities import (TemplateKind, ...)`. I don't think it's important that we type the entire package/module path when we don't have to. It's just as explicit where these classes come from because the import gives that information. I'm confident that any person who is used to Python would know to look here for that.